pkg/devfile/parser/sourceAttribute.go

```diff
@@ -37,50 +39,74 @@ func AddSourceAttributesForTemplateSpecContent(sourceImportReference v1.ImportRe
 //  to all elements of parent override that support attributes.
-func AddSourceAttributesForParentOverride(sourceImportReference v1.ImportReference, parentoverride *v1.ParentOverrides) {
-	for idx, component := range parentoverride.Components {
+func addSourceAttributesForParentOverride(sourceImportReference v1.ImportReference, parentOverrides *v1.ParentOverrides) {
+	for idx, component := range parentOverrides.Components {
 		if component.Attributes == nil {
-			parentoverride.Components[idx].Attributes = attributes.Attributes{}
+			parentOverrides.Components[idx].Attributes = attributes.Attributes{}
 		}
-		parentoverride.Components[idx].Attributes.PutString(ImportSourceAttribute, fmt.Sprintf("parentOverrides from: %s", resolveImportReference(sourceImportReference)))
+		parentOverrides.Components[idx].Attributes.PutString(ParentOverrideAttribute, resolveImportReference(sourceImportReference))
 	}
-	for idx, command := range parentoverride.Commands {
+	for idx, command := range parentOverrides.Commands {
 		if command.Attributes == nil {
-			parentoverride.Commands[idx].Attributes = attributes.Attributes{}
+			parentOverrides.Commands[idx].Attributes = attributes.Attributes{}
 		}
-		parentoverride.Commands[idx].Attributes.PutString(ImportSourceAttribute, fmt.Sprintf("parentOverrides from: %s", resolveImportReference(sourceImportReference)))
+		parentOverrides.Commands[idx].Attributes.PutString(ParentOverrideAttribute, resolveImportReference(sourceImportReference))
 	}
-	for idx, project := range parentoverride.Projects {
+	for idx, project := range parentOverrides.Projects {
 		if project.Attributes == nil {
-			parentoverride.Projects[idx].Attributes = attributes.Attributes{}
+			parentOverrides.Projects[idx].Attributes = attributes.Attributes{}
 		}
-		parentoverride.Projects[idx].Attributes.PutString(ImportSourceAttribute, fmt.Sprintf("parentOverrides from: %s", resolveImportReference(sourceImportReference)))
+		parentOverrides.Projects[idx].Attributes.PutString(ParentOverrideAttribute, resolveImportReference(sourceImportReference))
 	}
-	for idx, project := range parentoverride.StarterProjects {
+	for idx, project := range parentOverrides.StarterProjects {
 		if project.Attributes == nil {
-			parentoverride.StarterProjects[idx].Attributes = attributes.Attributes{}
+			parentOverrides.StarterProjects[idx].Attributes = attributes.Attributes{}
 		}
-		parentoverride.StarterProjects[idx].Attributes.PutString(ImportSourceAttribute, fmt.Sprintf("parentOverrides from: %s", resolveImportReference(sourceImportReference)))
+		parentOverrides.StarterProjects[idx].Attributes.PutString(ParentOverrideAttribute, resolveImportReference(sourceImportReference))
 	}
 
 }
 
-// AddSourceAttributesForPluginOverride adds an attribute 'library.devfile.io/imported-from=<source reference>'
+// addSourceAttributesForPluginOverride adds an attribute 'library.devfile.io/imported-from=<source reference>'
```

Comment:
need to update func desc for the updated attribute key, same for the parent override func

Reply:
updated